docs/source/using/addons.adoc

```diff
@@ -161,6 +161,23 @@ $ minishift addons enable anyuid
 $ minishift addons disable anyuid
 ----
 
+[[apply-addons]]
+== Applying add-ons
+
+Add-ons can be explicitly executed with the link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
+You can use _add-on_ _apply_ command for both enabled and disabled add-ons. For applying multiple add-ons at once please specify multiple add-on names separated by space.
+
+_Tip_: To execute an add-on during link:../command-ref/minishift_start{outfilesuffix}[`minishift start`] you need to enable it using link:../command-ref/minishift_addons_enable{outfilesuffix}[`minishift addons enable`] command.
+
```

Comment:
@LalatenduMohanty I think this information should go as the first line of enabling and disabling add-ons. It is a necessary action for the Add-on functionality.

I suggest starting the Enabling and disabling add-ons section with the sentence on line #170 and adding this as the second sentence: Enabled add-ons automatically get executed during link:../command-ref/minishift_start{outfilesuffix}[`minishift start`].
This would set the context for the Apply part.